docs/examples/actions/basic.vue

```
@@ -0,0 +1,32 @@
+    key: 'retry',
+    label: 'Retry',
+    icon: RefreshRight
+  },
+  {
+    key: 'copy',
+    label: 'Copy',
+    icon: DocumentCopy
+  },
+]
+
+const onClick = ({ keyPath }: { keyPath: string[]  }) => {
```

ESLint

[error] Top-level functions should be declared with function keyword

(antfu/top-level-function)